style.css

```diff
@@ -0,0 +1,255 @@
+@import url("https://fonts.googleapis.com/css2?family=Alumni+Sans+Collegiate+One&family=Montserrat:wght@100;500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Alumni+Sans+Collegiate+One&family=Crete+Round&family=Montserrat:wght@100;500&display=swap");
+
+* {
+  margin: 0;
+
+  /* border: 1px solid black; */
+}
+
+/* class to import */
+.white {
+  color: white;
+}
+
+.blue-1 {
+  color: #091e42;
+}
+
+.blue-2 {
+  color: #091e42;
+}
+
+.yelow {
+  color: #ff6b00;
+}
+
+.bg-btn {
+  background-color: rgba(255, 255, 255, 0.24);
+}
+
+.font-montesori {
+  font-family: "Montserrat", sans-serif;
+}
+
+/* Content of the page */
+.nav-container {
+  display: flex;
+  width: auto;
+  height: auto;
+  align-items: center;
+  background-color: #3c3a39;
+  justify-content: space-between;
+  padding: 20px;
+  position: sticky;
+  z-index: 1;
+  top: 0;
+  font-family: "Montserrat", sans-serif;
+}
+
+.menu-btn {
+  background-color: unset;
+  border: none;
+}
+
+.logo {
+  color: white;
+  text-decoration: none;
+}
+
+.headline-container {
+  background-color: #1c1c19;
+  background-image: url(./assets/images/Illustration.png);
+  background-position: right bottom;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  padding: 100px 20px 20px 20px;
+  height: 60vh;
+  gap: 10px;
+  position: relative;
+  font-family: "Montserrat", sans-serif;
+}
+
+.headline-title {
+  color: #ff6b00;
+  font-size: 50px;
+  font-style: unset;
+  font-family: "Crete Round", serif;
+}
+
+.social-media-icones-container {
+  display: flex;
+  list-style: none;
+  margin-left: -3em;
+}
+
+/* today */
+#works-container {
+  display: grid;
+  grid-template-rows: 100px auto 450px 450px 450px 100px;
+}
+
+.recent-works-title-block {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: "Crete Round", serif;
+  font-size: 30px;
+}
+
+.recent-works {
+  display: flex;
+  flex-direction: column;
+  padding: 0 20px 20px 20px;
+  gap: 15px;
+}
```

Comment:
- [Optional] kindly, be sure that your design stick to the Figma, your design hasn't been left and right merged as the Figma one. 😍😍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/205422815-95434491-86fb-43d8-9941-405f2c1407a5.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/205422818-82eb930e-8403-44ec-82bd-f722beb6ef1d.png" width="200">|